experimental/framework/fluid-static/src/types.ts

```diff
@@ -71,3 +71,75 @@ export interface ContainerSchema {
      */
     dynamicObjectTypes?: LoadableObjectClass<any>[];
 }
+
+/**
+ * Base interface to be implemented to fetch each service's audience. The generic C allows consumers to further
+ * extend the client object with service-specific details about the connecting client, such as device information,
+ * environme
+ */
+export interface IServiceAudience<M extends IMember> {
+    /**
+     * Returns an map of all users currently in the Fluid session where key is the userId and the value is the
+     * member object
+     */
+    getMembers(): Map<string, M>;
+
+    /**
+     * Returns the current active user on this client once they are connected. Otherwise, returns undefined.
+     */
+    getCurrentMember(): M | undefined;
```

Comment:
I don't think "Current" implies the right thing here.  Maybe `getSelf()` or something?

Reply:
Updated to be getMyself and getMyClient respectively!